### DISCUSSION TITLE
Adding global font variable to all components in the page

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#159] [Discuss: New Feature Ideas]
While importing a font, should we provide the option to apply to all components at once?

**Comment by cruxcode** [comment]
Yes, this is an important feature. Does the capability to apply styles to body component sounds like a good option?

**Reply by darshitac11** [reply]
Yes!

**Reply by Meghsham15** [reply]
yes sounds good

**Comment by darshitac11** [comment]
Issue has been created! #433
